server/src/test/java/io/unitycatalog/server/sdk/tables/SdkTableCRUDTest.java

```diff
@@ -59,7 +58,11 @@ public void testListTablesWithNoNextPageTokenShouldReturnNull() throws Exception
         createCommonResources();
         TableInfo testingTable = createDefaultTestingTable();
         ListTablesResponse resp = localTablesApi.listTables(testingTable.getCatalogName(), testingTable.getSchemaName(), 100, null);
-        assert resp.getNextPageToken() == null;
-        assert Objects.equals(resp.getTables(), List.of(testingTable));
+        assertThat(resp.getNextPageToken()).isNotNull();
+        assertThat(resp.getTables()).hasSize(1)
+                .first()
+                .usingRecursiveComparison()
```

Comment:
this seems to be a change in the test logic. @ravivj-db could you double check this?

Reply:
this is because previously the `assert` wasn't executed and the condition was actually never met (the lists weren't equal)